src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Sidenav from './component/sidenav/sidenav';
+import Events from './component/pages/events';
+import Reservation from './component/pages/reservation';
+import Reserve from './component/pages/reserve';
+import CreateEvent from './component/pages/new_event';
+import DeleteEvent from './component/pages/delete_event';
+import Login from './component/pages/login';
+import Signup from './component/pages/signup';
+import EventsDetails from './component/pages/events_details';
+
+function App() {
```

Comment:
## Reservations functionality

- [ ] At the time of this review, I can't reserve events.  I get the "This event has not been reserved" most of the time.  If I continue trying to reserve events, sometimes one works.

- [ ] When I manage to reserve an event, "My reservation" page is still empty.
This shows up in the console:
![image](https://user-images.githubusercontent.com/57421823/213192637-e0269fbd-45e9-499c-9090-425429353dd6.png)


- [ ] When I try to reserve an event, I can't change the date.

Check this on your end and if these are a bugs, kindly fix it so your app works as intended. 👍 

_All issues tested on Chrome and Firefox._

Reply:
Hey @Meltrust glad you have found something we haven't seen

- on the reserve page a user can only reserve an event once if you try to reserve an event it will notify you with a response.
- the error `can't access property id` occurs when there is no user to fetch the data for my reservation
- on the date, we agreed to use the exact date of the reservation happening meaning at what time the user reserved the event. 

generally, the explanation I made is to make things. we will look into it and do our best to resolve it quickly and submit it again.